tools/cli/go-whisk-cli/commands/property.go

```diff
@@ -190,8 +190,7 @@ var propertyUnsetCmd = &cobra.Command{
                     map[string]interface{}{"ok": color.GreenString("ok:")}))
             if len(DefaultAuth) > 0 {
                 okMsg += fmt.Sprintf(
-                    wski18n.T("; the default value of {{.default}} will be used.\n",
-                        map[string]interface{}{"default": boldString(DefaultAuth)}))
+                    wski18n.T("; the default value will be used. Run 'wsk property get --auth' to see the new value.\n"))
```

Comment:
The unset property function does remove the current property setting, write and reload the property file.
During reload of the property file, defaults are set.
For `DefaultAuth` and `DefaultAPIHost` the default value is "".

If these default values never changes, the discussed info message can be removed for the two properties.
Updating code.